src/crate/crash/command.py

```diff
@@ -368,6 +368,8 @@ def _try_exec_cmd(self, line):
             if message:
                 self.logger.info(message)
             return True
+        else:
+	    self.logger.critical('Command unknown')
```

Comment:
better error message:

Unknown command '<command>.' Type `\q` for a full list of available commands.